test/e2e/mco_test.go

```diff
@@ -27,3 +31,59 @@ func TestClusterOperatorRelatedObjects(t *testing.T) {
 		t.Error("ClusterOperator.RelatedObjects should contain the MCO namespace")
 	}
 }
+
+func TestMastersSchedulable(t *testing.T) {
+	cs := framework.NewClientSet("")
+	schedulerCR, err := cs.ConfigV1Interface.Schedulers().Get("cluster", metav1.GetOptions{})
+	if err != nil {
+		t.Fatal("Error while listing scheduler CR")
+	}
+	schedulerCR.Spec.MastersSchedulable = true
+	if _, err = cs.ConfigV1Interface.Schedulers().Update(schedulerCR); err != nil {
+		t.Fatal("Error while updating scheduler CR")
```

Comment:
let's `Fatalf` here and elsewhere to log the error we got as well - e2e is failing right now and this is not providing further info about what's happening

Reply:
See also other changes to use `require.NoError(t, error)` which enforces that.

Reply:
Ya, I saw it too. Will do that but it's taking 1:30 hrs to fail. Let me try it locally first.